packages/app/src/server/crowi/index.js

```diff
@@ -372,7 +372,24 @@ Crowi.prototype.setupPassport = async function() {
 };
 
 Crowi.prototype.setupSearcher = async function() {
-  this.searchService = new SearchService(this);
+  const searchDelegatorLogger = loggerFactory('growi:service:search');
+
+  this.searchService = await new SearchService(this);
+
+  if (this.configManager.getConfig('crowi', 'app:elasticsearchReindexOnBoot')) {
```

Comment:
- Assign the returned value in  to a variable
- Check falsy first so you can return early